tests/disjoint_comm_demo/main.cpp

```diff
@@ -0,0 +1,63 @@
+#include "enrico/comm.h"
```

Comment:
Same question about this test -- did you want to keep this around?

Reply:
Deleted